### ISSUE TITLE
Search Result

### ISSUE THREAD

**Issue by sabbath83** [prika/UAE-Future#27]
The search result is not clear to the user, please we need to change its design to be more clear.

**Comment by prika** [comment]
@nuno3cmonteiro o que é preciso fazer? Passaste isto ao designer?

**Comment by nuno3cmonteiro** [comment]
Não. Não vale a pena, conseguem criar algo que simplesmente torne o texto mais visivel. O que eles se queixam é que não conseguem ler por causa da imagem. Na minha ideia imagino uma caixa que pode ter alguma transparência mas que permita ver a fonte melhor. Mas deixo a solução completamente ao vosso critério, o objectivo é conseguir ler-se melhor o texto.

**Comment by prika** [comment]
Então, eu colocaria numa página sem o banner de cima (estilo página de [dicionário](http://54.93.89.184/Dictionary)).

Quanto mais informações sobre os termos que foram encontrados melhor. Se for de áreas especificas do site era interessante coloca-la (por exemplo: "Ecosystem"), se for relacionado com categorias podiam-se colocar as tags (por exemplo 'News'). Imagens relacionadas com os resultados.

Depende do que conseguirem enviar pela programação. Sugiro a seguinte estrutura:

```
<div class="title">
         <h1>Resultados de Pesquisa</h1>
         <h2>Foram encontrados {{número de resultados}} para a pesquisa "{{texto pesquisado}}"</h2>
</div>


<section id="dictionaryResultsSection">
      <div class="container">
            <div class="row">
                 <div id="containerWords" class="col-md-6 containerWords">
                      <a id="" href="">{{Resultado1}} - {{Descrição ou texto relacionado com a página}}</a>
                      <a id="" href="">{{Resultado2}} - {{Descrição ou texto relacionado com a página}}</a>
                      <a id="" href="">{{Resultado3}} - {{Descrição ou texto relacionado com a página}}</a>
                 </div>
                 <div id="" class="col-md-6">
                      **Mais pesquisados/Sugestões de pesquisa/Contactos... ou algo do género...**
                 </div>    
            </div>
      </div>
</section>
```

**Comment by sabbath83** [comment]
Sofia temos um pequeno problema neste caso. Estamos a usar um controlo so Sitefinity que não é muito facil aplicar os css. O Filipe encontrou uma solução. Fica ao vosso critério (Nuno e Sófia).

http://54.93.89.184/uae_searchresults?searchQuery=Lorem&indexCatalogue=uae-index&wordsMode=0

**Comment by prika** [comment]
Mas não dá para não usar o bloco de html do `.fullBanner`?
Também posso forçar estilos na página mas preciso que coloquem um identificador tipo `id="searchPage"`

**Comment by sabbath83** [comment]
Acho que consegui aplicar o teu html. Só nao sei se deve ter imagem de fundo. Podes me dar uma ajuda?

**Comment by prika** [comment]
- [x] Rever estilos da paginação
- [ ] Colocar elemento do lado direito (nova search ou uma imagem) 
- [x] Retirar o limite de altura e o scroll
